scripts/lighthouse/fixtures.js

```diff
@@ -0,0 +1,67 @@
+const url = 'http://localhost:7080/news/articles/c9rpqy7pmypo';
+const config = {
+  urls: [url],
+  thresholds: {
+    accessibility: 0,
+    seo: 0,
+    pwa: 0,
+    performance: 0,
+    'best-practices': 0,
+  },
+  opts: {
+    chromeFlags: ['--headless'],
+  },
+};
+
+const validatedPassingScores = [
+  {
+    url,
+    scores: [
+      {
+        id: 'performance',
+        score: 1,
+        expectedScore: 0,
+        pass: true,
+      },
+      { id: 'pwa', score: 1, expectedScore: 0, pass: true },
+      { id: 'accessibility', score: 1, expectedScore: 0, pass: true },
+      {
+        id: 'best-practices',
+        score: 0.5,
+        expectedScore: 0,
+        pass: true,
+      },
+      { id: 'seo', score: 0.5, expectedScore: 0, pass: true },
+    ],
+  },
+];
+
+const validatedFailingScores = JSON.parse(
+  JSON.stringify(validatedPassingScores),
+);
+
+function createFailingScores(index) {
+  validatedFailingScores[0].scores[index].expectedScore = 1;
+  validatedFailingScores[0].scores[index].pass = false;
+}
+
+createFailingScores(3);
+createFailingScores(4);
```

Comment:
What's the purpose of lines 43-49? Feels like this was written to avoid CodeClimate shouting about duplication...?

Reply:
yes that's correct

Reply:
...in which case, feels like we've abstracted out some duplication for the sake of readability 🤔 I'd sooner have something like:

```js
const baseConfig = {
  ...bla
};

const fooBarTestCase = Object.assign(baseConfig, {
  foo: 'bar',
});

configs.push(fooBarTestCase); // etc
```

Happy to go ahead without refactor, just a bit of a head-scratcher on first-review!